svc/LocalStorageService.js

```diff
@@ -55,6 +63,21 @@ export class LocalStorageService {
     //------------------
     getInstance() {
         throwIf(!this._supported, 'Local Storage is not supported');
-        return store.namespace(XH.appName);
+        return store.namespace(this.getNamespace());
+    }
+
+    getNamespace() {
+        return `${XH.appCode}.${XH.getUsername().toLowerCase()}`;
```

Comment:
Technically we are requiring that HoistUser normalize usernames to lowercase:

https://github.com/exhi/hoist-core/blob/ff105dce5a563ba9ac27fb876a736449324012c8/src/main/groovy/io/xh/hoist/user/HoistUser.groovy#L22

Any reason to believe that mixed-case usernames are making it to the client?  I can understand the instinct to be defensive here and don't feel strongly about it, but that lowercase requirement is probably something we should take as a requirement / given throughout our client-side Hoist code.